example/get-media-crmls.rb

```diff
@@ -0,0 +1,38 @@
+require 'rets'
+
+client = Rets::Client.new({
+  login_url: 'url',
+  username: 'username',
+  password: 'password',
+  version: 'RETS/1.7.2' 
+})
+
+begin
+    client.login
+rescue => e
+    puts 'Error: ' + e.message
+    exit!
+end
+
+puts 'We connected! Lets get all the photos for a property...'
+
+
+photos = client.find (:all), {
+  search_type: 'Media',
+  class: 'Media',
+  query: '(ResourceRecordKeyNumeric=117562969),(MediaType=Image)'
```

Comment:
let's make the values here generic, similar to the property example here: https://github.com/estately/rets/blob/master/example/get-property.rb#L9